tests/methods/sa-gasaciscf-1/input.dat

```diff
@@ -34,14 +34,12 @@ set forte {
   gas1min             [0,0]
   gas1max             [2,1]
   avg_state           [[0,1,1],[2,3,1]]  
-  job_type            casscf #mcscf_two_step
```

Comment:
The `mcscf_two_step` code should reproduce `casscf` code by setting `CASSCF_MICRO_MAXITER 1`.

Reply:
I wonder if we should rename the keywords `CASSCF_...` to `MCSCF_...`.

Reply:
I think we should rename them. Thanks for suggesting this. I think this might be the right PR to do so.

Reply:
I am writing the documentation for this part of the code, so I'll take care of the renaming and making sure it's documented properly.